control_panel_api/management/commands/migrate_lambdas_data_1_buckets.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import os
+
+import boto3
+from django.core.management.base import BaseCommand
+
+from control_panel_api.management.commands.migrate_lambdas_data_utils import (
+    bucket_name,
+    is_eligible,
+)
+from control_panel_api.models import S3Bucket
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    NOTE: Needs permission to perform `iam:ListPolicies` action
+    """
+
+
+    help = "Add records for S3 buckets created by AWS Lambda functions."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--dry-run',
+            dest='dry_run',
+            help="Run command without side effects",
+            default=False,
+            action='store_true',
+        )
+
+    def handle(self, *args, **options):
+        iam = boto3.client('iam')
+        results = iam.list_policies(
+            Scope='Local',  # Customer managed policies only
+            OnlyAttached=True,  # Ignore non-attached policies
```

Comment:
This comment just repeats what the code says. Why ignore non-attached policies?

Reply:
Because we may have IAM policies for test S3 buckets not attached to anything. We don't need to create records for them in the CP and we can safely ignore those.